_posts/2020-03-05-grayskull.md

```diff
@@ -0,0 +1,331 @@
+# By the power of Grayskull... I have the Conda recipe!
+
+*The main goal of the Skeleto~~n~~r is to conquer Grayskull.*
+
+## Introduction
+
+All jokes aside, the new project [grayskull :skull:](https://github.com/marcelotrevisani/grayskull) was created with the __intention__ of generating better Conda recipes that would allow to package properly projects available in different channels such as PyPI, CRAN, Conan, GitHub register, GitHub repositories and so on. On top of that, Grayskull is also being developed to help [conda-forge](https://conda-forge.org/) to update recipes.
+
+
+## Current status
+
+Currently, Grayskull (version ``0.2.1``) is able to generate recipes just looking for packages on [PyPI](https://pypi.org/), and it is available on [PyPI](https://pypi.org/project/grayskull/) and [conda-forge](https://github.com/conda-forge/grayskull-feedstock).
```

Comment:
This is markdown, not rst, so you should only use single backticks rather than double